### PR TITLE
Removing old slides

### PR DESCRIPTION
**Pull request by ticidesign**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -34,11 +34,3 @@ As an organiser, you'll have to share the work done during a session with all of
 3. Clean the `katas/` folder. Only keep the katas you've done during the session and delete the others
 3. Create a release tag with the same name as your branch.
 4. Share it with the attendants.
-
-### Branch naming convention
-The branch name has to contain your chapter's name and the coding-dojo date like so: `melbourne/28-03-2018`. If your coding dojo session has more than one group, suffix the name with a letter from A to Z :
-```
```
Comment (ButenkoT) [inline CONTRIBUTING.md]: do we have a different folder for Melbourne now?

File: slides/index.html [hunk 2]
```diff
@@ -1,159 +1,115 @@
...
-<script type="text/javascript" src="js/highlight.pack.js"></script>
-<script type="text/javascript" src="js/app.js"></script>
-
-</body>
+<html>
+	<head>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+
+		<title>Coding Dojo - NodeGirls</title>
```
Comment (ButenkoT) [inline slides/index.html]: Muses Code JS

File: CONTRIBUTING.md (hunk 1 above)
Comment (ticidesign) [reply CONTRIBUTING.md]: Yes [here](https://github.com/muses-code-js/coding-dojo-melbourne)

File: slides/index.html (hunk 2 above)
Comment (ticidesign) [reply slides/index.html]: Done

**Pull request merged**